<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>465</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="notificationsGBox">
     <property name="title">
      <string>Notifications</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>361</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="nGBLayout">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetFixedSize</enum>
       </property>
       <item>
        <widget class="QCheckBox" name="showAddedCBox">
         <property name="text">
          <string>Show when device added</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showRemovedCBox">
         <property name="text">
          <string>Show when device removed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showMountedCBox">
         <property name="text">
          <string>Show when device mounted</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showUnmountedCBox">
         <property name="text">
          <string>Show when device unmounted</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="actionsGroupBox">
     <property name="title">
      <string>Actions</string>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>361</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="mountAddedCBox">
         <property name="text">
          <string>Mount on added</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showInternalCBox">
         <property name="text">
          <string>Show system disks</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="viewMountedCBox">
         <property name="text">
          <string>Execute view command when mounted</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="forceUnmountCBox">
         <property name="text">
          <string>Force unmount when busy</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Device format string:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="formatStringEdit">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>View command:        </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="viewCommandEdit">
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
